nemo/collections/cv/modules/data_layers/cifar10_datalayer.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from os.path import expanduser
+
+from torchvision.datasets import CIFAR10
+from torchvision.transforms import Compose, Resize, ToTensor
+
+from nemo.backends.pytorch.nm import DataLayerNM
+from nemo.core.neural_types import AxisKind, AxisType, ClassificationTargetType, ImageType, NeuralType
+from nemo.utils.decorators import add_port_docs
+
+__all__ = ['CIFAR10DataLayer']
+
+
+class CIFAR10DataLayer(DataLayerNM, CIFAR10):
+    """
+    A "thin DataLayer" -  wrapper around the torchvision's CIFAR10 dataset.
+    """
+
+    def __init__(
+        self, name=None, height=32, width=32, data_folder="~/data/cifar10", train=True, batch_size=64, shuffle=True
```

Comment:
please let's use Python3 type hints on user exposed classes, really helps understanding imho

Reply:
👍

Reply:
I generally agree - but please take a look once again at the list of arguments, _all of them_ come with default values, that clearly indicate the types ;)